pr-statistics-spent-time.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+
+# PARAMETERS ----------------------------
+GITHUB_REPO="status-im/status-dev-cli"
+DEBUG=1
```

Comment:
What does this do exactly? Can we turn it off by default? When I run script I see:

DEBUG -- url is https://api.github.com/repos/status-im/status-dev-cli/pulls?state=closed -- nb page : 1
DEBUG -- downloading page 1 ...
DEBUG -- page 1 downloaded
Min: 12
Q1: 22656
Mean: 245305
Q3: 7546970
Max: 10459664

Reply:
changed to a variable named VERBOSE. It allow to print some info while computing.
done in commit https://github.com/status-im/status-react/pull/2893/commits/0b287d778877586a7249552f78af30f2223027ac